_protocols/beer-dna-sequencing.md

```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
-3. Mix the prepared library gently by pipetting up and down prior to loading
-4. Add 75 μl of sample to the flow cell via the SpotON sample port in a dropwise fashion. Ensure each drop flows into the port before adding the next
-5. Gently replace the SpotON sample port cover, making sure the bung enters the SpotON port, close the priming port and the MinION lid
+3. Prepare a tube with 7.5 µl extracted DNA and 2.5 µl FRA
 
+    > FRA is fragmentation mix, designed to perform a random fragmentation of genomic DNAs into smaller pieces by enzyme treatments.
 
+4. Mix gently by flicking the tube, and spin down
+5. Put the tubes into a thermocycler for 1 min at 30°C and then 1 min at 80°C
 
-## Protocol: Software setup
+    > Denaturation of DNA, i.e. the process by which double-stranded deoxyribonucleic acid unwinds and separates into single-stranded strands through the breaking of hydrophobic stacking attractions between the bases, takes place at 80°C. The primers are first annealed to target sequence at 30°C. 
 
-### MinKNOW
+6. Put the tube on ice until next step
+7. Attach adapters by adding 1 µl of RAP into the tube
 
-MinKNOW is the program needed to connect your computer with MinION. It has a graphic user interface for configuring and running the sequencing. Furthermore, it has integrated base-caller software to convert the raw nanopore signals to the strings of nucleotides, in fastq format.
+    > RAP contains adapters that are ligated to the DNA. They are needed for the sequencing to guide the DNA fragments through the nanopores
 
-#### Installation:
+8. Mix gently by flicking the tube, and spin down
+9. Incubate the reaction for 5 min at room temperature
+10. Store the library on ice until ready to load
 
-- Please search for "MinION Software" in the Nanopore downloads web page https://community.nanoporetech.com/downloads.
-- Select your host operating system and follow the link to the installation instructions.
-- Since the supported systems and the installation procedure is constantly updated, please always refer to the community website for details.
+## Sequencing
 
-#### Starting the software:
+1. Prepare the reagents
+    1. Mix the SQB tube by vortexing, spin down and return to ice
+    2. Mix the FB tube by vortexing, spin down and return to ice
+    3. Spin down the FLT tube, mix by pipetting, and return to ice
+2. Prepare loading port
+    1. Open priming port on the flowcell to check for small bubbles
+    2. Remove possible bubbles by taking 200 µl liquid from the port using the P1000 pipette
+    3. Remove the liquid by turning the wheel of the pipet but only until 220-230 µl
 
-Requirements:
-- You have to have a account at nanopor MinION to start the software
-- The divice need to be attached to the computer to start the software
+    Note: removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times
```

Comment:
Given before it is written 200, I am not sure to understand how we can not remove more than 30 here